src/main/resources/citizen/templates/document/defendantResponseReceipt.html

```diff
@@ -739,41 +742,59 @@
 {% endif %}
 {% endif %}
 
+{% if mediation is defined %}
+<div class="section">
+  <div class="section-heading">
+    <strong>Mediation</strong>
+  </div>
+  <div class="section-body">
+    <table>
+      <tr>
+        <td><strong>Willing to try mediation</strong></td>
+        <td>
+          {% if mediation %}Yes{% else %}No{% endif %}
```

Comment:
this would be shorter? `{{ "Yes" if mediation else "No" }}`

Reply:
Nice, I saw a few variations on this

Reply:
Alas, disnae work
`Unexpected token of value "if" and type NAME, expected token of type PRINT_END`

Reply:
damn mustn't work in the java library :(

Reply:
But pebble doesn't support this notation; I'll go with a ternary
`{% mediation ? "Yes" : "No" %}`